### PR TITLE
add trans extrnsion

### PR DESCRIPTION
**Pull request by andrey1s**

### CODE REVIEW

File: Resources/config/twig_extensions.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="f_devs_locale.twig_extension.class">FDevs\LocaleBundle\Twig\TranslatorExtension</parameter>
+    </parameters>
+
+    <services>
+        <service id="f_devs_locale.twig_extension" class="%f_devs_locale.twig_extension.class%" public="false">
+            <argument>%f_devs_locale.translator_extensions%</argument>
+            <call method="setContainer"><argument type="service" id="service_container" on-invalid="null"/></call>
```
Comment (vmelnik-ukraine) [inline Resources/config/twig_extensions.xml]: @andrey1s please format nested elements properly

File: DependencyInjection/FDevsLocaleExtension.php [hunk 2]
```diff
@@ -26,6 +26,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\XmlFileLoader($container, new FileLocator([__DIR__ . '/../Resources/config', realpath(dirname($refl->getFileName()) . '/Resources/config')]));
 
         $container->setParameter($this->getAlias() . '.allowed_locales', $config['allowed_locales']);
+        $container->setParameter($this->getAlias() . '.translator_extensions', $config['translator_extensions']);
```
Comment (vmelnik-ukraine) [inline DependencyInjection/FDevsLocaleExtension.php]: @andrey1s it seems that you use `f_devs_locale.translator_extensions` param only in the private service `f_devs_locale.twig_extension`. What do you think about removing translator_extensions param from global params bag and inject it via definition instead?

**Comment by andrey1s** [comment]
done, do not forget to update the version.

**Comment by andrey1s** [comment]
done

File: Resources/config/twig_extensions.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="f_devs_locale.twig_extension.class">FDevs\LocaleBundle\Twig\TranslatorExtension</parameter>
+        <parameter key="f_devs_locale.translator_extensions" type="collection"></parameter>
+    </parameters>
+
+    <services>
+        <service id="f_devs_locale.twig_extension" class="%f_devs_locale.twig_extension.class%" public="false">
+            <argument>%f_devs_locale.translator_extensions%</argument>
```
Comment (vmelnik-ukraine) [inline Resources/config/twig_extensions.xml]: @andrey1s one little thing, it should be just `<argument />` now.

**Pull request merged**